k8s/kustomization.yaml

```
@@ -0,0 +1,13 @@
+kind: Kustomization
+
+resources:
+  - cnpg/cluster.yaml
+  - mongodb/deployment.yaml
+  - influxdb/deployment.yaml
+  - redis/deployment.yaml
+  - app/deployment.yaml
+
+images:
+  - name: first-rust-app
+    newTag: master
```

YAMLlint (1.37.1)

[error] no new line character at the end of file

(new-line-at-end-of-file)